commons/src/test/scala/dagr/commons/util/LoggerTest.scala

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
We do use it in tests, but in other packages, so it doesn't count towards coverage in this package!